tests/integration/profiles/five-safes-crate/test_5src_1_responsible_project.py

```diff
@@ -0,0 +1,170 @@
+        }
+        INSERT {
+            ?project schema:funding "Not a grant (literal replacement)" .
+        }
+        WHERE {
+            ?action a schema:CreateAction ;
+                    schema:agent ?agent .
+            ?agent schema:memberOf ?project .
+            ?project schema:funding ?grant .
+            ?grant a schema:Grant .
+        }
+        """
+    )
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
+        requirement_severity=Severity.REQUIRED,
+        expected_validation_result=False,
+        expected_triggered_requirements=["Responsible Project"],
+        expected_triggered_issues=[
+            "The property 'funding' of the Responsible Project MUST be of type schema:Grant."
+        ],
+        profile_identifier="five-safes-crate",
+        rocrate_entity_mod_sparql=sparql,
+    )
+
+
+def test_5src_responsible_project_member_not_organization():
+    """
+    Test a Five Safes Crate where a Responsible Project's `member` property
+    is NOT of type schema:Organization.
+    (We replace the member reference with a literal.)
+    """
+    sparql = (
+        SPARQL_PREFIXES
+        + """
+        DELETE {
+            ?project schema:member ?org .
+        }
+        INSERT {
+            ?project schema:member "Not an organization (literal replacement)" .
+        }
+        WHERE {
+            ?action a schema:CreateAction ;
+                    schema:agent ?agent .
+            ?agent schema:memberOf ?project .
+            ?project schema:member ?org .
+            ?org a schema:Organization .
+        }
+        """
+    )
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
+        requirement_severity=Severity.REQUIRED,
+        expected_validation_result=False,
+        expected_triggered_requirements=["Responsible Project"],
+        expected_triggered_issues=[
+            "The property 'member' of the Responsible Project MUST be of type schema:Organization."
+        ],
+        profile_identifier="five-safes-crate",
+        rocrate_entity_mod_sparql=sparql,
+    )
+
+
+# ---- MAY warns tests
+
+
+def test_5src_responsible_project_missing_funding_property():
+    """
+    Test a Five Safes Crate where a Responsible Project does NOT have the `funding` property.
+    This should trigger the SHACL info: 'The Responsible Project does not have the property `funding`.'
+    """
+    sparql = (
+        SPARQL_PREFIXES
+        + """
+        DELETE {
+            ?project schema:funding ?f .
+        }
+        WHERE {
+            ?action a schema:CreateAction ;
+                    schema:agent ?agent .
+            ?agent schema:memberOf ?project .
+            ?project schema:funding ?f .
+        }
+        """
+    )
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
+        requirement_severity=Severity.OPTIONAL,
+        expected_validation_result=False,  # or True if Info is not treated as failure
```

Comment:
think this comment is no longer needed as the tests are working! (as long as `requirement_severity=Severity.OPTIONAL` anyway)
```suggestion
        expected_validation_result=False,
```

Reply:
Done.